### ISSUE TITLE
Re-examine asset pipeline

### ISSUE THREAD

**Issue by techgique** [CDRH/orchid#37]
It seems I misunderstood how parts of the asset pipeline are working. I thought I tested this, but Sass files added to Orchid's `app/assets/stylesheets/orchid/` directory are raising pre-compilation errors when named in `@ext_css`.

In addition, it seems regardless of what contents are present in `app/assets/javascripts/orchid.js`, as long as the main app calls `//= require orchid` from its `applications.js`, adding `orchid/search` to `@ext_js` still works and _doesn't_ raise a pre-compilation error.

Much confuse. In the mean time, foregoing conditional CSS files for Orchid and adding to `orchid.scss`.

**Comment by jduss4** [comment]
When I switched to using the sassc gem locally, I got an error about the global css:

`ActionView::Template::Error (File to import not found or unreadable: global/*.`

**Comment by techgique** [comment]
`sassc-rails` fix it?

**Comment by jduss4** [comment]
yeah, I messed up by just putting `sassc` and not `sassc-rails`, that's my bad!  I was using the ruby library, specifically:  https://github.com/sass/sassc-ruby#readme

Whoops!

**Comment by jduss4** [comment]
In general we are going to want to look this over for Rails 6 changes, anyway!
